rom/laghos_rom.cpp

```diff
@@ -2272,28 +2260,21 @@ void ROM_Operator::UndoInducedGramSchmidt(const int var, Vector &S, bool keep_da
 
 void ROM_Operator::InducedGramSchmidtInitialize(Vector &S)
 {
-    InducedGramSchmidt(1, S); // velocity
-    InducedGramSchmidt(2, S); // energy
-    basis->ComputeReducedMatrices();
-    if (useReducedMv)
-    {
-        ComputeReducedMv();
-        ComputeReducedMe();
-    }
-
-    if (hyperreduce)
+    if (useGramSchmidt && !sns1)
```

Comment:
Now this function is called even if `useGramSchmidt` is false. Would it be more accurate to rename the function as something like `SetupReducedMatrices`?

Reply:
The routine is now renamed as `ApplyHyperreduction`.